packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -205,49 +205,49 @@ LogEvent objects are read-only. The following properties are available:
-| client                | The client that requested an action                                                    | [Client object](#client-object)                                 | TRUE     | FALSE  | TRUE     |           |           |
-| request               | The request that initiated an action                                                   | [Request object](#request-object)                               | TRUE     | FALSE  | TRUE     |           |           |
+| actor                 | Describes the entity that performs an action                                          | [Actor object](#actor-object)                                   | TRUE     | FALSE  | TRUE     |           |           |
+| client                | The client that requests an action                                                    | [Client object](#client-object)                                 | TRUE     | FALSE  | TRUE     |           |           |
+| request               | The request that initiates an action                                                   | [Request object](#request-object)                               | TRUE     | FALSE  | TRUE     |           |           |
 | outcome               | The outcome of an action                                                               | [Outcome object](#outcome-object)                               | TRUE     | FALSE  | TRUE     |           |           |
 | target                | Zero or more targets of an action                                                      | Array of [Target object](#target-object)                        | TRUE     | FALSE  | TRUE     |           |           |
 | transaction           | The transaction details of an action                                                   | [Transaction object](#transaction-object)                       | TRUE     | FALSE  | TRUE     |           |           |
 | debugContext          | The debug request data of an action                                                    | [DebugContext object](#debugcontext-object)                     | TRUE     | FALSE  | TRUE     |           |           |
 | authenticationContext | The authentication data of an action                                                   | [AuthenticationContext object](#authenticationcontext-object)   | TRUE     | FALSE  | TRUE     |           |           |
 | securityContext       | The security data of an action                                                         | [SecurityContext object](#securitycontext-object)               | TRUE     | FALSE  | TRUE     |           |           |
 
-> The actor and/or target of an event is dependent on the action performed. All events have actors but not all have targets.
+>**NOTE:** The actor and/or target of an event is dependent on the action that is performed. All events have actors but not all have targets.
```

Comment:
Microsoft Manual of Style says to avoid "and/or" usage; if "or" works alone, I suggest this change.